internal/controller/pipelinerollout_controller_test.go

```diff
@@ -947,17 +954,19 @@ func Test_processExistingPipeline_PPND(t *testing.T) {
 
 		t.Run(tc.name, func(t *testing.T) {
 
-			// first delete Pipeline in case it already exists, in Kubernetes
-			_ = numaflowClientSet.NumaflowV1alpha1().Pipelines(defaultNamespace).Delete(ctx, defaultPipelineRolloutName, metav1.DeleteOptions{})
+			// first delete Pipeline and PipelineRollout in case they already exist, in Kubernetes
+			_ = numaflowClientSet.NumaflowV1alpha1().Pipelines(defaultNamespace).Delete(ctx, defaultPipelineName, metav1.DeleteOptions{})
 
 			pipelineList, err := numaflowClientSet.NumaflowV1alpha1().Pipelines(defaultNamespace).List(ctx, metav1.ListOptions{})
 			assert.NoError(t, err)
 			assert.Len(t, pipelineList.Items, 0)
 
-			// create Pipeline definition
 			rollout := createPipelineRollout(tc.newPipelineSpec)
+			_ = numaplaneClient.Delete(ctx, rollout)
+
 			rollout.Status.Phase = tc.initialRolloutPhase
 			if tc.initialInProgressStrategy != nil {
+				rollout.Status.UpgradeInProgress = *tc.initialInProgressStrategy
 				rollout.Status.UpgradeInProgress = *tc.initialInProgressStrategy
```

Comment:
is this a duplicate line?

Reply:
ah yeah! Thanks for catching, must be from the rebase.